openc3/bin/openc3cli

```diff
@@ -433,7 +458,9 @@ def cli_pkg_install(filename, scope:)
     if File.extname(filename) == '.gem'
       OpenC3::GemModel.install(filename, scope: scope)
     else
-      OpenC3::PythonPackageModel.install(filename, scope: scope)
+      process_name = OpenC3::PythonPackageModel.install(filename, scope: scope)
+      print "Installing..."
+      wait_process_complete_internal(process_name, scope: scope)
```

Comment:
How did this fix the python install? It wasn't waiting for complete and then the process died before it was done?

Reply:
This cleaned up leaving process status as running forever, because the process that was monitor complete had died.